src/executables/util_cs.c

```diff
@@ -3286,7 +3286,8 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 	{
 	  db_clear_host_connected ();
 
-	  printf ("\n *** Applied Info. *** \n");
+	  fprintf (stdout, "\n *** Applied Info. *** \n");
```

Comment:
Because, printf (...) is same with fprintf (stdout, ...), you can delete this line.
For example,

```
printf ("\n *** Applied Info. *** \n");
flush (stdout);
```

Reply:
Thank you. I will test and apply it.